java/code/src/com/redhat/rhn/frontend/struts/StrutsDelegate.java

```diff
@@ -341,7 +341,7 @@ public Date readDatePicker(DynaActionForm form, String name, int yearDirection)
         //if it is not there, then that means we should always evaluate the
         //date picker.  Otherwise, we evaluate if it tells us to do so.
         if (!form.getMap().containsKey(DatePicker.SCHEDULE_TYPE) ||
-                form.get(DatePicker.SCHEDULE_TYPE) == null ||
+                StringUtils.isEmpty(String.valueOf(form.get(DatePicker.SCHEDULE_TYPE))) ||
```

Comment:
If i understand this correctly this change will now make it so that if schedule type is an empty string it will use the date picker and also since `String.valueOf(null)` returns the string "null" it will not use the date picker anymore when schedule type is null. Is this correct?